runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineJob.java

```diff
@@ -167,20 +182,19 @@ public DataflowPipelineJob getReplacedByJob() {
    * @throws InterruptedException
    */
   @Nullable
+  @VisibleForTesting
   public State waitToFinish(
```

Comment:
This is used by TestDataflowRunner. Even though it is not strictly test code, it is for test purposes.